src/lib/storage/index/group_key/variable_length_key_store.hpp

```diff
@@ -137,7 +137,7 @@ class VariableLengthKeyStore {
 
     void increment() { _data += _key_alignment; }
     void decrement() { _data -= _key_alignment; }
-    void advance(ChunkOffset n) { _data += n * _key_alignment; }
+    void advance(size_t n) { _data += n * _key_alignment; }
```

Comment:
Change `n`?

Reply:
I think it is one of the cases where we it's super obvious what `n` is about and we are actually extending the boost facade, which uses `n`. I would keep it like that.